x-pack/plugins/security/public/management/roles/edit_role/privileges/kibana/privilege_form_calculator/privilege_form_calculator.ts

```diff
@@ -47,11 +47,12 @@ export class PrivilegeFormCalculator {
    *
    * @param featureId the feature id to get the Primary Feature KibanaPrivilege for.
    * @param privilegeIndex the index of the kibana privileges role component
+   * @param allSpacesSelected indicates if the privilege form is configured to grant access to all spaces.
    */
   public getDisplayedPrimaryFeaturePrivilegeId(
     featureId: string,
     privilegeIndex: number,
-    allSpacesSelected?: boolean
+    allSpacesSelected: boolean
```

Comment:
__note:__ to make sure we don't miss optional arguments like this one in the future, I made it required.